src/component/mxgraph/overlay/OverlayConverter.ts

```diff
@@ -33,7 +34,38 @@ export class OverlayConverter {
     ['middle-right', { horizontalAlign: 'right', verticalAlign: 'middle' }],
   ]);
 
-  convertPosition(overlay: Overlay): MxGraphCustomOverlayOptions {
+  convert(overlay: Overlay): MxGraphCustomOverlayOptions {
+    const position = this.convertPosition(overlay);
+    const style = this.convertStyle(overlay);
+    return { position, style };
+  }
+
+  private convertPosition(overlay: Overlay): MxGraphCustomOverlayPosition {
     return this.overlayPositions.get(overlay.position);
   }
+
+  private convertStyle(overlay: Overlay): MxGraphCustomOverlayStyle {
+    const defaultStyle = { fill: { color: StyleDefault.DEFAULT_FILL_COLOR.valueOf() }, stroke: { color: StyleDefault.DEFAULT_STROKE_COLOR.valueOf() } };
```

Comment:
⚠️ move this out of the class as it is a constant

Reply:
<img width="1126" alt="image" src="https://user-images.githubusercontent.com/4921914/114401550-513f2780-9ba3-11eb-8d77-072a2086c53d.png">

Reply:
I had in mind to put the constant in the OverlayConverter.ts file and not make it a variable of the enum for now